### PR TITLE
Rewrite README

### LINKED ISSUE #22: Maintain and update outdated contents

### PR DESCRIPTION
**Pull request by boundless-forest**
Fix #22

### CODE REVIEW

**Review by nblogist — APPROVED** [review]
Thank you for this @boundless-forest

**Comment by nblogist** [comment]
I don't have write access to this @hackfisher

**Review by hackfisher — APPROVED** [review]

**Comment by hackfisher** [comment]
> I don't have write access to this @hackfisher

The branch protection rule require at least one approval, I noticed that you give one approval, but I don't why it have no effects.

Add you as the maintainer of this repo anyway.

**Pull request merged**
